tests/integration/dbclient/appl/src/test/java/io/helidon/tests/integration/dbclient/appl/it/health/HealthCheckIT.java

```diff
@@ -150,11 +148,11 @@ public void testHealthCheckWithCustomNamedQuery() {
      */
     @Test
     public void testHealthCheckWithCustomQuery() {
-        LOGGER.info(() -> "Running test testHealthCheckWithCustomQuery");
+        LOGGER.log(Level.INFO, () -> "Running test testHealthCheckWithCustomQuery");
```

Comment:
This kind of statements is not really useful. The testing framework should let us know this is happening (this is a general comment for any test in this PR).

Reply:
Yes, this is there from the beginning, because tests were called also trough application server and in that case I had no information about what's going on there.
But all tests are already working properly so I'm removing such a messages in whole `integration/dbclient` source tree.
Also will replace all loggers with System.Logger